phpstorm/CheckBuildUpdate.ps1

```diff
@@ -44,6 +53,8 @@ Write-Host "Update nuspec"
 $nuspec_template.package.metadata.version = $newVersion
 if (Test-Path "$directory\phpstorm.nuspec") { Remove-Item "$directory\phpstorm.nuspec" }
 $nuspec_template.save("$directory\phpstorm.nuspec")
+Write-Host "Update nuspec <releaseNotes> with new URL"
+(Get-Content "$directory\phpstorm.nuspec") -replace('{{release_url}}', $release_url) | Set-Content "$directory\phpstorm.nuspec"
```

Comment:
I've removed the version "variable" and the whole element from the nuspec template file and made the PowerShell script create and insert it instead. This looks a lot cleaner to me. I originally left the variable in the template file because of problems with creating new XML elements. (I actually thought you had to go through instantiating a .NET object first and stuff like that, but it's actually relatively easy)